### ISSUE TITLE
Documentation does not have right color contrast/ readability when using dark reader

### ISSUE THREAD

**Issue by jkodu** [gatsbyjs/gatsby#12592]
Using the plugin
- https://chrome.google.com/webstore/detail/dark-reader/eimadpbcbfnmbkopoojfekhnkhdbieeh?hl=en

I use the plugin Dark Reader, and it does not seem to play well for gatsby documentation pages:

![image](https://user-images.githubusercontent.com/20978252/54430776-82c33b80-471c-11e9-951e-a0a07f4c09fb.png)

**Comment by pieh** [comment]
I would think that this is issue more with this browser extension and not with gatsby website directly. I have no idea how this extension works and what's needed on our website to make it play nice with it

**Comment by pieh** [comment]
It seems like this extension doesn't support SPAs that loads extra styles after initial load.

**Comment by pieh** [comment]
Also - there is open issue to provide toggle on gatsbyjs.org that would switch between default light theme and dark theme - https://github.com/gatsbyjs/gatsby/issues/8789, which would solve this better than generic browser extension (IMO)

**Comment by jkodu** [comment]
I have tested the extension on several SPA, and it works fine.

But if they're is a roadmap to toggle styles that's cool with me.

**Comment by pieh** [comment]
> I have tested the extension on several SPA, and it works fine.

But do those SPA add extra styles? Or just load one stylesheet? Gatsby does code split and lazy load styles and from few checks I did it seem like this browser extension does handle inital style (inverting etc), but does not handle any styles that are added after that
